aws/table_aws_elasticache_update_action.go

```diff
@@ -0,0 +1,174 @@
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "engine",
+				Description: "The ElastiCache engine (Redis or Memcached).",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "estimated_update_time",
+				Description: "The estimated length of time for the update to complete.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "nodes_updated",
+				Description: "The progress of the service update on the replication group.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "service_update_name",
+				Description: "The unique ID of the service update.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "service_update_recommended_apply_by_date",
+				Description: "Recommended date to apply the update for compliance.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "service_update_release_date",
+				Description: "The date the update was first available.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "service_update_severity",
+				Description: "Severity level of the service update.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "service_update_status",
+				Description: "Current status of the service update.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "service_update_type",
+				Description: "Type of service update (security update, etc).",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "sla_met",
+				Description: "Indicates if nodes were updated by recommended date.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "update_action_available_date",
+				Description: "Date the update became available to the replication group.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "update_action_status",
+				Description: "Current status of the update action.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "update_action_status_modified_date",
+				Description: "Last modification date of the update action status.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+
+			// Standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.From(extractElastiCacheUpdateActionId),
+			},
+		}),
+	}
+}
+
+func listElastiCacheUpdateActions(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	client, err := ElastiCacheClient(ctx, d)
+
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_elasticache_update_action.listElastiCacheUpdateActions", "client_error", err)
+		return nil, err
+	}
+	// Limiting the results
+	maxLimit := int32(50)
+	if d.QueryContext.Limit != nil {
+		limit := int32(*d.QueryContext.Limit)
+		if limit < maxLimit {
+			if limit < 1 {
```

Comment:
Could we remove the check when the value is less than 1? Steampipe automatically handles this scenario by stopping execution if the limit value is below 1.